### PR TITLE
Per current ancestry.com documentation, added Email 2FA flag

### PR DESCRIPTION
**Pull request by betsythefc**

### CODE REVIEW

File: _data/other.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ websites:
     img: ancestry.png
     tfa:
       - sms
+      - email
     doc: https://support.ancestry.com/s/article/MFA
     exception: "2FA is currently only available for customers with a U.S. based phone number."
```
Comment (phallobst) [inline _data/other.yml]:
```suggestion
    exception: "SMS 2FA is currently only available for customers with a U.S. based phone number."
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: _data/other.yml (hunk 1 above)
Comment (betsythefc) [reply _data/other.yml]: I re-read the documentation, and it doesn't look like you need to contact customer support to you email:

The steps are under the heading "What to do if you don’t receive the code", and I was able to enable it without contacting customer support.

Comment (betsythefc) [reply _data/other.yml]: I agree with putting SMS in front of that exception.

Comment (phallobst) [reply _data/other.yml]: At the top of the doc page, it says
> If you don't have a mobile phone, you can enable it for email by giving us a call.

Is this information outdated?

Comment (betsythefc) [reply _data/other.yml]: That's interesting. It must require the mobile phone first to enable email delivery.

Comment (betsythefc) [reply _data/other.yml]: It looks like this information is out of date. Without a phone number, I was able to enable 2FA, and it gave me the option to select a phone number or an email.

Comment (phallobst) [reply _data/other.yml]: Okay, I updated my suggestion accordingly.

**Review by phallobst — APPROVED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**